packages/back-end/src/models/ExperimentSnapshotModel.ts

```diff
@@ -219,6 +221,25 @@ export async function updateSnapshot({
   });
   if (!experimentSnapshotModel) throw "Internal error";
 
+  const shouldUpdateExperimentAnalysisSummary =
+    experimentSnapshotModel.type === "standard" &&
+    experimentSnapshotModel.status !== "running";
```

Comment:
do these statuses work for bandits as well? SRM is similar but computed slightly differently, ME should be identical, but the other power-calculator-related statuses perhaps not...

Reply:
I think it's computed differently but we still only return one p-value and one status, so I think the changes here should work.

Reply:
@lukebrawleysmith recommended to skip the mid-experiment power calculation for Bandits, so `health.power` will be optional.

@lukesonnet So it sounds that this implementation works for SRM & ME for Bandit Experiments?

Reply:
I think it should.